### PR TITLE
Feature/141 make employees table sortable

### PR DESCRIPTION
**Pull request by tjanowsk**
Replaced HTML table with ag-grid

Styling still needs some tweaking.
new_employee_table

### CODE REVIEW

File: frontend/src/components/employees/index.tsx [hunk 1]
```diff
@@ -158,12 +129,22 @@ const Employees = () => {
...
-  const openDeleteModal = (staff: StaffData) => () => {
+  const openDeleteModal = (staff: StaffData) => {
     setDeleteModalState(true);
     setSelectedStaff(staff);
   };
 
-  const tableColumns = makeTableColumns(editUser, openDeleteModal);
+  const cellClicked = (event: any) => {
+    switch (event.colDef.field) {
+      case 'editbtn':
```
Comment (connorsettle) [inline frontend/src/components/employees/index.tsx:139]: might as well pull it out as const (within this component file)

File: frontend/src/components/employees/index.tsx [hunk 2]
```diff
@@ -178,8 +159,15 @@ const Employees = () => {
     <div className={employeesStyle.employees}>
       <h1 className={employeesStyle.header}> Employees </h1>
       {addButton}
-      <div className={employeesStyle.tableContainer}>
-        {renderTable(tableColumns)(staffList)}
+      <div
+        className={[employeesStyle.tableContainer, 'ag-theme-alpine'].join(
```
Comment (connorsettle) [inline frontend/src/components/employees/index.tsx:163]: Can just interpolate it

Comment (tjanowsk) [reply frontend/src/components/employees/index.tsx:163]: Can you explain what you mean by 'interpolate' here?

Comment (connorsettle) [reply frontend/src/components/employees/index.tsx:159]:
```
`${employeesStyle.tableContainer} ag-theme-alpine`
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

**Pull request merged**